interfaces/policy/helpers.go

```diff
@@ -221,6 +221,36 @@ func checkSlotConnectionConstraints(connc *ConnectCandidate, cstrs []*asserts.Sl
 	return firstErr
 }
 
+func checkSnapTypeSlotInstallationConstraints1(ic *InstallCandidateMinimalCheck, slot *snap.SlotInfo, cstrs *asserts.SlotInstallationConstraints) error {
+	if err := checkSnapType(slot.Snap, cstrs.SlotSnapTypes); err != nil {
+		return err
+	}
+	if err := checkOnClassic(cstrs.OnClassic); err != nil {
+		return err
+	}
+	return nil
+}
+
+func checkMinimalSlotInstallationConstraints(ic *InstallCandidateMinimalCheck, slot *snap.SlotInfo, cstrs []*asserts.SlotInstallationConstraints) (bool, error) {
+	var firstErr error
+	var hasSnapTypeConstraints bool
+	// OR of constraints
+	for _, cstrs1 := range cstrs {
```

Comment:
I'm bad with names, but these seem rather poor to me. Maybe use the full name at this point? Certainly
```go
for _, constraint := range constraints {
```
would be clearer

Reply:
@mvo5 for better or worse this was following the naming conventions of the rest of the file, I don't think we should change them just in 1 function. I would prefer to keep this way here and change everything possibly in a follow up